docs/disable-parameterized-tests.adoc

```diff
@@ -0,0 +1,250 @@
+		"fine disable me all you want", // 5
+		"not those one, though!" // 6
+	}
+)
+void single(String reason) {
+	// ...
+}
+----
+
+Using both `matches` and `contains` in a single annotation is no longer permitted as of {VERSION_OR_DATE_PLACHOLDER}.
+
+== DisableIfParameter
+
+This extension can be used to selectively disable parameterized tests based on their parameter values (converted with `toString()`).
+The extension comes with three annotations, covering different use-cases:
+
+- `@DisableIfAnyParameter`, non-repeatable
+- `@DisableIfAllParameters`, non-repeatable
+- `@DisableIfParameter`, repeatable
+
+The annotations are only supported on test method level for parameterized tests.
+Unlike the `@Disabled` API provided in JUnit Jupiter, which disables the test on first encounter of the annotation, the extension evaluates each execution of a parameterized test.
+As a consequence, instead of disabling the entire set of parameterized tests, each test is possibly disabled individually.
+All three annotations require that you specify one of two parameters, `contains` *or* `matches`.
+`@DisableIfAnyParameter` will disable test executions if *any* argument either contains or matches any of the given strings.
+`@DisableIfAllParameters` will disable test executions if *all* arguments either contain or match any of the given strings.
+`@DisableIfParameter` will disable test executions if a *specified* argument either contains or matches any of the given strings.
+
+=== Using `contains`
+
+[source,java]
+----
+@DisableIfAllParameters(contains = "the")
+@ParameterizedTest
+@CsvSource(value = {
+		"If the swift moment I entreat:;Tarry a while! You are so fair!",
+		"Then forge the shackles to my feet,;Then I will gladly perish there!",
+		"Then let them toll the passing-bell,;Then of your servitude be free,",
+		"The clock may stop, its hands fall still,;And time be over then for me!"
+}, delimiter = ';')
+void disableAllContains(String line, String line2) {
+}
+----
+
+The test `disableAllContains` ordinarily would run four times, but the second execution gets disabled because both arguments contain "the" (the second argument as part of "there").
+Using the same test with a different annotation would look like this:
+
+[source,java]
+----
+@DisableIfAnyParameter(contains = "Then")
+@ParameterizedTest
+@CsvSource(value = {
+		"If the swift moment I entreat:;Tarry a while! You are so fair!",
+		"Then forge the shackles to my feet,;Then I will gladly perish there!",
+		"Then let them toll the passing-bell,;Then of your servitude be free,",
+		"The clock may stop, its hands fall still,;And time be over then for me!"
+}, delimiter = ';')
+void disableAnyContains(String line, String line2) {
+}
+----
+
+The test `disableAnyContains` ordinarily would run four times, but the second and third executions get disabled because an argument contains "Then".
+The last execution does not get disabled, because the extension is case-sensitive.
```

Comment:
I would add (yes add, not move) the information about case-sensetive to the general information about the disableIfParameter too, cause important.

So the general information and here an example for it